java/src/jmri/jmrix/dccpp/DCCppPacketizer.java

```diff
@@ -132,8 +138,8 @@ public boolean portReadyToSend(jmri.jmrix.AbstractPortController p) {
     @Override
     protected void loadChars(jmri.jmrix.AbstractMRReply msg, java.io.DataInputStream istream) throws java.io.IOException {
         int i;
-        StringBuilder m = new StringBuilder();
-        log.debug("loading characters from port");
+        StringBuilder m = new StringBuilder("");
```

Comment:
No need to change. But `new StringBuilder()` and `new StringBuilder("")` does exactly the same thing.
https://docs.oracle.com/javase/7/docs/api/java/lang/StringBuilder.html#StringBuilder(java.lang.String)

Reply:
Interesting, I don't recall touching the StringBuilder line at all......

Reply:
I think somebody did the opposite change not far ago. I'm wondering if this change reverts that earlier change.

The change doesn't matter, but since I think somebody else did the opposite, probably to clean up code, I think it would be good if you removed this change.

Reply:
No problem, done.